Fix `declaration-property-value-no-unknown` and other false positives for multiline SCSS interpolation (#7406)
stylelint/stylelint: .changeset/new-bananas-sell.md

---
"stylelint": patch
---

> Fixed: Ignore multiline SCSS interpolation e.g. in `declaration-property-value-no-unknown`

Reviewer: ```suggestion
Fixed: `declaration-property-value-no-unknown` and other false positives for multiline SCSS interpolation.
```

To follow our conventions.
